airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/xmin/XminCtidUtils.java

```diff
@@ -99,6 +100,7 @@ private static List<ConfiguredAirbyteStream> identifyNewlyAddedStreams(final Con
     final Set<AirbyteStreamNameNamespacePair> newlyAddedStreams = new HashSet<>(Sets.difference(allStreams, alreadySeenStreams));
 
     return fullCatalog.getStreams().stream()
+        .filter(c -> c.getSyncMode() == SyncMode.INCREMENTAL)
```

Comment:
I guess we already filter these out downstrem (in PostgresCtidHandler and PostgresXminHandler). Any reason we want to do this here as well?